service-python/pdfgenerator/src/lib/templates/hypertension-v2.html

```diff
@@ -1,467 +1,489 @@
 <!doctype html>
 <html lang="en">
 
-<head>
-  <!-- Required meta tags -->
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1"/>
-  <meta name="pdfkit-orientation" content="Landscape"/>
-  <meta name="pdfkit-footer-right" content="Page [page] of [topage]"/>
-  <meta name="pdfkit-footer-font-name" content="NotoSans"/>
-  <meta name="pdfkit-footer-font-size" content="8"/>
-  <meta name="pdfkit-page-size" content="Legal"/>
-  <meta name="pdfkit-margin-left" content="0.5in"/>
-  <meta name="pdfkit-margin-right" content="0.5in"/>
-  <meta name="pdfkit-margin-top" content="0.5in"/>
-  <meta name="pdfkit-margin-bottom" content="0.5in"/>
-  <meta name="pdfkit-zoom" content="1.33"/>
-  <meta name="pdfkit-dpi" content="72"/>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta name="pdfkit-orientation" content="Landscape"/>
+    <meta name="pdfkit-footer-right" content="Page [page] of [topage]"/>
+    <meta name="pdfkit-footer-font-name" content="NotoSans"/>
+    <meta name="pdfkit-footer-font-size" content="8"/>
+    <meta name="pdfkit-page-size" content="Legal"/>
+  <!-- 40px -->
+    <meta name="pdfkit-margin-left" content="0.555in"/>
+  <!-- 80px -->
+    <meta name="pdfkit-margin-right" content="1.1in"/>
```

Comment:
The page size got larger, I think from this change

Reply:
Yep, I was trying to fix the DPI issue which required to redo the sizing but looks like that did more harm than good. Redid the PR to only make the main updates